### DISCUSSION TITLE
Will there be support for quantum key exchange and quantum digital signature?

### DISCUSSION THREAD

**Discussion by Chewhern** [jedisct1/libsodium#1081] [Ideas]
It seems to be a question that is not appropriate to ask now, but does libsodium planned to add support for quantum resistant key exchange and quantum resistant digital signature in the future once it's safe and secure to use(after announced by NIST)?

**Comment by jedisct1** [comment]
This is very likely, yes.

**Reply by Chewhern** [reply]
Alright good to know
